### ISSUE TITLE
[Bug]: A11y violation with Calendar Contrast

### ISSUE THREAD

**Issue by Malvin95** [Malvin95/NLCC-FoodHub#58]
## 🐞 Summary

Storybook is flagging an issue wherein the 'colour-contrast' is an issue on the calendar. 

## 🔁 Steps to Reproduce

While running the storybook app **AND** lines 33-42 inside the ```history-dashboard.stories.tsx``` have been removed.

1. Go to /pages/HistoryDashboard
2. Click on any test that is not a skeleton test.
3. Look at the tray at the bottom of the screen and click the **Accessibility** Tab, and look at either the *Violations* or *Inconclusive* tabs.
4. Observe error ```The color contrast between text and its background meets WCAG AA contrast ratio.```
---

## ✅ Expected Behavior

This should not be flagged.

## ❌ Actual Behavior

The flag is apparent.

## 🌐 Environment

Storybook local app.

## 🧾 Screenshots / Logs

<img width="3386" height="1834" alt="Image" src="https://github.com/user-attachments/assets/3392e800-94c6-409a-a66b-7ae438e17dd4" />

## 🗒️ Additional Notes

Optional but useful.
